include/Dolphin/gx.h

```diff
@@ -138,6 +138,16 @@ typedef enum _GXIndTexWrap {
 	GX_IND_TEX_WRAP_0,
 } GXIndTexWrap;
 
+typedef enum _GXTexOffset {
+	/* 0x0 */ GX_TO_ZERO,
+	/* 0x1 */ GX_TO_SIXTEENTH,
+	/* 0x2 */ GX_TO_EIGHTH,
```

Comment:
is this also from TP? are we using this comment convention anywhere else?

Reply:
Yep, these were from TP - forgot to integrate our comment style with them, but fixed it! ty for spotting it!